docs/IE_DG/network_state_intro.md

```diff
@@ -209,9 +209,131 @@ Decsriptions can be found in [Samples Overview](./Samples_Overview.md)
+
+**TensorFlow:** *BlockLSTM* will be converted to TensorIterator operation, TensorIterator body will contain LSTM Cell operation, Peepholes, InputForget modifications are not supported.
+*While* layer will be converted to TensorIterator, TensorIterator body can contain any supported operations, but dynamic cases, when count of iterations can’t be calculated in shape inference (ModelOptimizer conversion) time are not supported.
+
+**TensorFlow2:** *While* layer will be converted to Loop operation. Loop body can contain any supported operations.
+
+**Kaldi:** Kaldi models will already contain Assign/ReadValue (Memory) operations after model conversion. TensorIterator/Loop operations never be generated.
+
+## LowLatencу2
+
+LowLatency transformation changes the structure of the network containing [TensorIterator](../ops/infrastructure/TensorIterator_1.md) and [Loop](../ops/infrastructure/Loop_5.md) by adding the ability to work with the state, inserting the Assign/ReadValue layers as it is shown in the picture below.
+
+#### Example of applying LowLatency2 transformation:
+![applying_low_latency_2_example](./img/applying_low_latency_2.png)
+
+After applying the transformation, ReadValue operations can receive other operations as an input, as shown in the picture above. These inputs should set the initial value for initialization of ReadValue operations. However, such initialization is not supported in the current State API implementation. Input values are ignored and the initial values for the ReadValue operations are set to zeros unless otherwise specified by the user via [State API](#openvino-state-api).
+
+### Steps to apply LowLatency Transformation
+
+1. Get CNNNetwork. Either way is acceptable:
+
+	* [from IR or ONNX model](./Integrate_with_customer_application_new_API.md)
+	* [from nGraph Function](../nGraph_DG/build_function.md)
+
+2. Change the number of iterations inside TensorIterator/Loop nodes in the network using [Reshape](ShapeInference.md) feature. 
+
+For example, *sequence_lengths* dimension of input of the network > 1, it means TensorIterator layer will have number_of_iterations > 1. We can reshape the inputs of the network to set *sequence_dimension* to exactly 1.
+
+```cpp
+
+// Network before reshape: Parameter (name: X, shape: [2 (sequence_lengths), 1, 16]) -> TensorIterator (num_iteration = 2, axis = 0) -> ...
+
+cnnNetwork.reshape({"X" : {1, 1, 16});
+
+// Network after reshape: Parameter (name: X, shape: [1 (sequence_lengths), 1, 16]) -> TensorIterator (num_iteration = 1, axis = 0) -> ...
+	
+```
+**Unrolling**: If the LowLatency2 transformation is applied to a network containing TensorIterator/Loop nodes with exactly 1 iteration inside, then these nodes will be unrolled, otherwise, the nodes remain as they are. Please see [the picture](#example-of-applying-lowlatency2-transformation) for more details.
```

Comment:
```suggestion
**Unrolling**: If the LowLatency2 transformation is applied to a network containing TensorIterator/Loop nodes with exactly one iteration inside, these nodes are unrolled; otherwise, the nodes remain as they are. Please see [the picture](#example-of-applying-lowlatency2-transformation) for more details.
```